components/feature/downloads/src/main/java/mozilla/components/feature/downloads/AbstractFetchDownloadService.kt

```diff
@@ -360,6 +364,33 @@ abstract class AbstractFetchDownloadService : Service() {
         downloadedFile.delete()
     }
 
+    /**
+     * Adds a file to the downloads database system, so it could appear in Downloads App
+     * (and thus become eligible for management by the Downloads App) only for compatible devices
+     * otherwise nothing will happen.
+     */
+    @VisibleForTesting
+    internal fun addToDownloadSystemDatabaseCompact(download: DownloadState) {
+        if (SDK_INT <= Build.VERSION_CODES.P) {
```

Comment:
I think it reads better as `< Q`, since Q introduces the new download system that causes issues.

Reply:
👍

Reply:
Fixed :)